tasks/docker_py_install.yml

```diff
@@ -1,16 +1,29 @@
 ---
 - block: # Install docker-py
+  - name: Install dependency
+    package:
+      name: 
+      - python-setuptools
+      - epel-release
+      state: present
+  
   - name: Check to see if pip is already installed.
     command: pip --version
     ignore_errors: true
     changed_when: false
     when: install_docker_py
     register: install_docker_pip_is_installed
 
-  - name: Install pip if not installed
-    shell: curl https://bootstrap.pypa.io/get-pip.py | python
+  - name: Install pip
+    package:
+      name: "{{ 'python-pip' if ansible_python.version.major == 2 else 'python3-pip' }}"
+      state: latest
     when: install_docker_py and install_docker_pip_is_installed.rc != 0
 
+  # - name: Install pip if not installed
```

Comment:
Remove this entirely or add a comment that says this is deprecated and retain this for debugging purpose.